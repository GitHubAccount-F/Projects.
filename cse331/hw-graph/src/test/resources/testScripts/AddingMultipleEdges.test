#Tests to see if multiple edges can be added between two nodes

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

#Adds nodes to graphs
AddNode graph1 one
AddNode graph1 two
AddNode graph2 one
AddNode graph2 two
AddNode graph3 one
AddNode graph3 two

#Adds the edge
AddEdge graph2 one two e1
AddEdge graph3 one two e1
AddEdge graph3 one two e2

#Check to see if edge was added
ListChildren graph1 one
ListChildren graph2 one
ListChildren graph3 one