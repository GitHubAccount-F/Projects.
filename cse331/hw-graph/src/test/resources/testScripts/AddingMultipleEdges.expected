#Tests to see if multiple edges can be added between two nodes

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3

#Adds nodes to graphs
added node one to graph1
added node two to graph1
added node one to graph2
added node two to graph2
added node one to graph3
added node two to graph3

#Adds the edge
added edge e1 from one to two in graph2
added edge e1 from one to two in graph3
added edge e2 from one to two in graph3

#Check to see if edge was added
the children of one in graph1 are:
the children of one in graph2 are: two(e1)
the children of one in graph3 are: two(e1) two(e2)